CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## UNRELEASED
+## 0.48.0 (September 01, 2022)
 
 FEATURES:
```

Comment:
could we add to breaking changes that support for k8s 1.19 and 1.20 is dropped

Reply:
Also the breaking change for `Kubernetes 1.24 Support` says "Kubernetes 1.24+" but we don't support anything above 1.24 yet.

Reply:
Good call, thanks! How does that look now?